### PR TITLE
[DA-403] Synthetic PPI generation based on CSV spec

### PR DESCRIPTION
**Pull request by danqrodney**
Changing import_questionnaires.sh to pull latest questionnaires from github.

### CODE REVIEW

File: rest-api/data_gen/fake_participant_generator.py [hunk 1]
```diff
@@ -574,12 +602,69 @@ def _choose_name(self, answer_map):
...
+      return [{'valueDateTime': (min_date +
+                                 datetime.timedelta(seconds=random.randint(0, seconds_diff)))
+                                 .isoformat()}
+              for _ in xrange(answer_count)]
+    if int(answer_spec['boolean_answer_count']) > 0:
+      return [{'valueBoolean': random.random() < 0.5}]
+    if int(answer_spec['string_answer_count']) > 0:
+      return [{'valueString': ''.join([random.choice(string.lowercase) for _ in xrange(20)])}
+              for _ in xrange(answer_count)]
+    if int(answer_spec['uri_answer_count']) > 0:
```
Comment (jmandel) [inline rest-api/data_gen/fake_participant_generator.py]: Can we make this example.com or something else that obviously won't point to data?

File: rest-api/data_gen/fake_participant_generator.py [hunk 2]
```diff
@@ -574,12 +602,69 @@ def _choose_name(self, answer_map):
...
+      if random.random() > float(num_participants) / float(self._answer_specs_max_participants):
+        continue
+      num_questionnaire_responses = int(answer_spec['num_questionnaire_responses'])
+      num_answers = int(answer_spec['num_answers'])
+      answer_count = 1
+      if num_answers > num_questionnaire_responses:
+        rand_val = random.random() * num_answers / num_questionnaire_responses
+        if rand_val > 1:
+          # Set the answer count to 2 or more
+          answer_count = int(1 + min(1, rand_val))
```
Comment (jmandel) [inline rest-api/data_gen/fake_participant_generator.py]: The min function here always returns 1, no?

File: rest-api/data_gen/fake_participant_generator.py (hunk 1 above)
Comment (danqrodney) [reply rest-api/data_gen/fake_participant_generator.py]: Done.

File: rest-api/data_gen/fake_participant_generator.py (hunk 2 above)
Comment (danqrodney) [reply rest-api/data_gen/fake_participant_generator.py]: Oops. Changed to max.

**Comment by danqrodney** [comment]
I had to push with -f, so the old stuff is gone. Responded to your comments.

**Pull request merged**